Assets/AltTester/Editor/AltTester.AltTesterUnitySDK.Editor.asmdef

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "AltTester.AltTesterUnitySDK.Editor",
```

Comment:
For sake of simplicity of code review purposes **collapse everything within `Assets/AltTester/`** and/or mark the filse as `**Viewed**`.